About.html

```diff
@@ -0,0 +1,163 @@
+        <h2 class="main-logo-header default-family fs-24 fw-700 fc-darkgrey mb-1">Meksel Festival</h2>
+        <span class="primary-bdb mb-1"></span>
+        <h3 class="main-logo-secundary-header default-family fs-16 fw-500 fc-greyish mb-1">Food prepaparation for Meksel Festival
+           based on the culture of Gurage People" </h3>
+      </div>
+      <img class="logo-main" src="../Resource/Images/events/13.jpg" alt="meskel day" />
+    </section>
+    <section id="main-past-events" class="main-past-events d-flex d-column d-align-center d-justify-center">
+      <h2 class="main-events-header default-family fs-24 fw-700 fc-darkgrey mb-1 text-center">See the past Meksel Events
+      </h2>
+      <span class="primary-bdb mb-1"></span>
+      <h3 class="main-events-secundary-header default-family fs-16 fw-500 fc-greyish mb-1">Take a look at the previous
+        Events!</h3>
+      <div class="past-events-img-container d-flex d-justify-center d-wrap">
+        <div class="overlay-container d-flex d-column d-justify-center d-align-center relative mt-1">
+          <img class="past-event-img past-event-filter" src="../Resource/Images/events/2022.jpg" alt="meskel" />
+          <div class="img-overlay d-flex d-column d-align-center d-justify-center absolute">
+            <h2 class="default-family fs-24 fw-700 fc-white text-center relative">2022</h2>
+            <h3 class="default-family fs-16 fw-700 fc-white text-center relative mt-1">Meksel Festival - 2022</h3>
+          </div>
+        </div>
+        <div class="overlay-container d-flex d-column d-justify-center d-align-center relative mt-1">
+          <img class="past-event-img past-event-filter" src="../Resource/Images/events/2023.jpg" alt="day meskel" />
+          <div class="img-overlay d-flex d-column d-align-center d-justify-center absolute">
+            <h2 class="default-family fs-24 fw-700 fc-white text-center relative">2023</h2>
+            <h3 class="default-family fs-16 fw-700 fc-white text-center relative mt-1">Meksel Festival - 2023</h3>
+          </div>
+        </div>
+
+      </div>
+    </section>
+  </main>
+  <section id="partners-section" class="partners-section d-none-desk">
+    <div id="partners-container" class="partners-container d-flex d-column d-align-center">
+      <h2 class="default-family fs-24 fw-700 fc-white mb-1">Partners</h2>
+      <span class="primary-bdb mb-1"></span>
+      <div id="partners-company-container"
+        class="partners-company-container d-inline-flex desk-w d-column d-align-center d-justify-center">
+        <img class="partner-logo mt-2" src="../Resource/Images/partners/gurage.png" alt="partner1" />
+        <img class="partner-logo mt-2" src="../Resource/Images/partners/disney+_logo.png" alt="partner1" />
+        <img class="partner-logo mt-2" src="../Resource/Images/partners/prime_logo.png" alt="partner2" />
+        <img class="partner-logo mt-2" src="../Resource/Images/partners/universal_logo.png" alt="partner3" />
```

Comment:
- [ ] For the project requirements, kindly make sure the `Partners` design on the `About` page sticks as much as possible the original design for Behance e.g: height, size, aignment,...

| Template Design | Your Design |
| ---------------------- | ------------------ |
| ![Screenshot from 2023-04-27 00-54-01](https://user-images.githubusercontent.com/83042742/234720329-58cd7d61-92dd-4be5-8496-3bb143262c74.png) | ![Screenshot from 2023-04-27 00-59-39](https://user-images.githubusercontent.com/83042742/234720490-4c2a66e9-bb46-4506-91b7-a89b7eb6a94e.png) |